### DISCUSSION TITLE
Not able to delete the metadata from the destination org

### DISCUSSION THREAD

**Discussion by Vaibhasrivastava** [scolladon/sfdx-git-delta#332] [Q&A]
I've pushed some destructive changes but the deploy command is not deleting the metadata from the destination org, even after action is succesful.

**Comment by scolladon** [comment]
It should be routed to [sfdx cli](https://github.com/salesforcecli/sfdx-cli) right ?

**Reply by Vaibhasrivastava** [reply]
It is sfdx force:mdapi:deploy -d destructiveChanges/destructiveChanges.xml -u SIT --ignorewarnings
I've used this command

**Reply by scolladon** [reply]
Do you have errors if you remove the --ignorewarnings flag?

**Reply by Vaibhasrivastava** [reply]
Checking will push the screenshot

**Reply by scolladon** [reply]
@Vaibhasrivastava do you have news to share?
